packages/thumbprint-react/components/InputRow/__snapshots__/test.jsx.snap

```diff
@@ -212,7 +212,7 @@ exports[`works with \`Input\` and \`Button\` 1`] = `
         }
       }
     >
-      <Input
+      <ForwardRef
```

Comment:
Can we add a `.displayName` property to the component to keep the display name?

https://reactjs.org/docs/forwarding-refs.html#displaying-a-custom-name-in-devtools